frontend/Github/GithubIssue.svelte

```diff
@@ -49,8 +52,32 @@
     export let aggregated = false;
 
     let deleting = false;
+    let showRuns = false;
     let issueState = issue.state;
 
+    let resolvedRuns;
+
+
+    const resolveRuns = async function(runs) {
```

Comment:
I didn't look into code, but how about other approach: when querying runs for the issue - why not to get required build_id and built_number along? So we don't need to have reslveRuns at all?

Reply:
Will have to do a migration and adjust the table, which will require code like this anyway, so we can do that as a follow-up to speed this up.

Reply:
only if needed. I was curious only.

Reply:
Potentially the current approach can be slow, as we are doing N queries for N runs (as using indices for a query like this is actually slower and gets extra data), one thing we could do here is to use `execute_async` and resolve those futures after queuing all of them, but from what I've checked the current approach is fast enough for up to 100 runs per issue.